qa/L0_metrics/histogram_metrics_test.py

```diff
@@ -165,14 +171,29 @@ def check_existing_metrics(model_name, wait_secs_per_req, delta):
 
             # Test non-decoupled model metrics
             non_decoupled_model_count = get_histogram_metric_key(
-                first_response_family, non_decoupled_model_name, "1", "count"
+                FIRST_RESPONSE_HISTOGRAM, non_decoupled_model_name, "1", "count"
             )
             non_decoupled_model_sum = get_histogram_metric_key(
-                first_response_family, non_decoupled_model_name, "1", "sum"
+                FIRST_RESPONSE_HISTOGRAM, non_decoupled_model_name, "1", "sum"
             )
             self.assertNotIn(non_decoupled_model_count, histogram_dict)
             self.assertNotIn(non_decoupled_model_sum, histogram_dict)
 
+    def test_buckets_override(self):
```

Comment:
Do you have a test to see what happens when you pass some invalid or non-existent family in the new `config.pbtxt` metric_control fields?

Some examples:
1. Specify valid `nv_inference_first_response_histogram_ms` but histograms are disabled so it shouldn't be present -- what happens?
2. Specify invalid non-existent metric `abc123` - what happens?

Reply:
Looks like these might be in `unknown_metric_family` tests in `L0_model_config`, but double checking

Reply:
It looks like (1) might still be a valid case to test, because it looks like this metric is in the `metric_map_` by default [here](https://github.com/triton-inference-server/core/blob/e550d6b915ec8efd11bead0b0fc2e3d739c9d3ef/src/metric_model_reporter.h#L86-L87), so this [check](https://github.com/triton-inference-server/core/blob/e550d6b915ec8efd11bead0b0fc2e3d739c9d3ef/src/metric_model_reporter.cc#L85) will always pass.

Looks like it might be harmless - but sharing in case it would be unexpected to hit this code path if the metric family didn't actually exist (ex: with histograms disabled).

Reply:
Case 1 is fine. I can add a test case to make sure the server produces no errors.

Reply:
Added test for valid model_metrics config with histogram disabled